docs/Course Wiki/MATH Course Offerings/MATH257.md

```diff
@@ -19,26 +19,23 @@ MATH257 (Linear Algebra with Computational Applications) is a 3-credit-hour clas
 MATH257 is the first linear algebra course that covers many fundamental concepts in linear algebra. It starts with linear equations and linear systems, and introducing the concept of vectors and matrices to solve linear systems. The course then delves into matrix operations such as addition and multiplication, and the different types of matrices such as elementary and inverse matrices. All of these concepts are regularly utilized as you proceed onto more difficult topics such as subspaces, eigenvectors, linear transformations, and orthogonal projections, so make sure you are comfortable with the fundamentals.
 
 ## Prerequisites
- - [MATH 221](MATH221.md)
+ - [MATH221](MATH221.md). Calculus is used in the course.
```

Comment:
Move the commentary to under the bullet point, and say something like "[MATH221](MATH221.md) is the only official prerequisite for this course.  It is important to be familiar with calculus before taking MATH257, as it is used in the course."